src/controllers/product.js

```diff
@@ -235,11 +235,56 @@ const orderRequest = async (req, res) => {
   }
 };
 
+// eslint-disable-next-line consistent-return
+const approveProduct = async (req, res) => {
```

Comment:
maybe we can change naming as approveRequest or approveRequester, it is giving the function main focus with that kind of naming i guess

Reply:
Ok changed it